src/mobile.js

```diff
@@ -0,0 +1,49 @@
+/*
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+/**
+ * Setup the B&A participant servers for mobile ads.
+ *
+ * This file contains the logic for the testing app,
+ * and does not contain Protected Audience logic.
+ */
+
+import fs from 'fs';
+import https from 'https';
+
+import sspA from './participants/mobile/ssp-a/server.js';
+import dspC from './participants/mobile/dsp-c/server.js';
+
+const DSP_C_PORT = 5001;
```

Comment:
Let's put this on 5005.  It doesn't really matter in terms of running it, but in terms of maintainability for us, I would like each participant to have a separate port.